docs/Bed_Mesh.md

```diff
@@ -372,14 +372,19 @@ are identified in green.
 
 ### Adaptive Meshes
 
-Adaptive bed meshing is a way to speed up the bed mesh generation by only probing
-the area of the bed used by the objects being printed. When used, the method will
-automatically adjust the mesh parameters based on the area occupied by the defined
-print objects.
+#### Concept:
+
+Adaptive bed meshing is a way to speed up the bed meshing process as well as waste less mesh information by only probing
```

Comment:
What does "waste less mesh information" mean? What information is being wasted if adaptive is not being used?

Also, a nit: can you please conform to the document's line length?

Reply:
> What does "waste less mesh information" mean? What information is being wasted if adaptive is not being used?
Well, if you're doing a regular full mesh for a small cube on a big bed, you're really only using 1 probed point and some interpolation rather than all of the probed points fitting to the object being printed.

> Also, a nit: can you please conform to the document's line length?

I don't know what I'm doing here, I'm just trying to help out.